portfolio/src/main/webapp/script.js

```diff
@@ -131,16 +131,25 @@ function fadeInIntro() {
  */
 function getComments() {
   const maxComments = document.getElementById('max-comments').value;
-  if (maxComments === null) maxComments = 5; /* Show at most 5 comments by default. */
+  if (maxComments === null) maxComments = 5; /* Show at most 5 comments */
 
-  fetch(`/comments?max-comments=${maxComments}`).then(response => response.json()).then((comments) => {
+  fetch(`/comments?max-comments=${maxComments}`)
+  .then(response => response.json())
+  .then((response) => {
     const commentsContainer = document.getElementById('comments-container');
     while (commentsContainer.firstChild) { /* Remove old comments */
       commentsContainer.removeChild(commentsContainer.firstChild);
     }
-    comments.forEach((comment) => {
-      commentsContainer.appendChild(createCommentElement(comment));
-    });
+
+    if(response.success === true) {
```

Comment:
Can you use the original response to check for 'ok' or the status instead of passing this in the JSON message?

Reply:
Good point. you can use response.ok to check status (https://developer.mozilla.org/en-US/docs/Web/API/Response/ok). here is an example - http://screen/uZT1SJWG4uA